sbt-airframe/src/sbt-test/sbt-airframe/generate-client/server/src/main/scala/myapp/server/MyServer.scala

```diff
@@ -31,36 +31,36 @@ object MyServer extends LogSupport {
 
     d.build[FinagleClient] { finagleClient =>
       val client = new myapp.spi.ServiceClient(finagleClient)
-      val future = client.myService.hello(100).map { v =>
+      val future = client.MyService.hello(100).map { v =>
```

Comment:
This will be a breaking change for existing http clients generated by sbt-airframe (cc: @takezoe @shimamoto @Pritesh-Patel )

Reply:
Yes. For us, conductor's Scala.js code will need to be fixed after upgrading to airframe 20.8.0.